ci-operator/step-registry/ipi/conf/aws/oidc-creds-provision/ipi-conf-aws-oidc-creds-provision-commands.sh

```diff
@@ -16,8 +16,34 @@ export XDG_RUNTIME_DIR="${HOME}/run"
 export REGISTRY_AUTH_PREFERENCE=podman # TODO: remove later, used for migrating oc from docker to podman
 mkdir -p "${XDG_RUNTIME_DIR}"
 
-oc registry login
-oc adm release extract --credentials-requests --cloud=aws --to="/tmp/credrequests" "$RELEASE_IMAGE_LATEST"
+echo "RELEASE_IMAGE_LATEST: ${RELEASE_IMAGE_LATEST}"
+
+build_pull_secret="/tmp/build_pull_secret"
+oc registry login --to=${build_pull_secret}
+
+# Merge two pull secret file
+#
+# for build triggered by Prow job, build0x's secret from ${build_pull_secret} is required.
+# for build triggered by Prow API, registry.ci.openshift.org and quay.io's secrets from ${CLUSTER_PROFILE_DIR}/pull-secret is required.
+
+new_pull_secret="/tmp/new_pull_secret"
+curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o /tmp/jq && chmod +x /tmp/jq
```

Comment:
maybe we can update the ref image from "cli: latest" to "cli-jq: latest" to skip the jq download.

Reply:
Oh, probably would not work, because a payload image does not have a `cli-jq` image, `cli-jq` only exists in prow ci.

Reply:
let me try.